packages/react/src/themes/teams/components/Status/statusStyles.ts

```diff
@@ -35,16 +35,33 @@ const getTextColor = (state: string, variables: StatusVariables) => {
   }
 }
 
-const statusStyles: ComponentSlotStylesInput<StatusPropsWithDefaults, StatusVariables> = {
+export const sizeToPxValue = {
```

Comment:
Can we introduce a type there, i.e.:

```suggestion
export const sizeToPxValue: Record<SizeValue, number> = {
```